pkg/file/joiner/joiner.go

```diff
@@ -31,6 +31,18 @@ func NewSimpleJoiner(getter storage.Getter) file.Joiner {
 }
 
 func (s *simpleJoiner) Size(ctx context.Context, address swarm.Address) (dataSize int64, err error) {
+	toDecrypt := len(address.Bytes()) == (swarm.HashSize + encryption.KeyLength)
+
+	var addr []byte
```

Comment:
we need a different name here. we already have the `address` param, having `addr` on top of this makes things confusing.

i think line `45` also supports this change